bin/db-checker

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const spawn = require('child_process').spawn;
+const leveldown = require('leveldown');
+
+//200Mb is the maximun allowed size for the manifest file.
+//It is better to run compaction when the file is bigger than this.
+const MAX_MANIFEST_SIZE = 200 * 1024 * 1024;
+
+if (require.main === module) {
+  const dbLocation = process.argv[2];
+  const manifestFile = fs.readdirSync(dbLocation)
+                         .find(f => f.match(/MANIFEST\-\d*/));
+
+  if (fs.statSync(path.join(dbLocation, manifestFile)).size > MAX_MANIFEST_SIZE) {
```

Comment:
I guess compactation might not be able to get an smaller enough size at some point, would be great to add some kind of warning in that case + instruction on what to do, because if this fails + the database fails to open and the service is killed next time the same will happen... so maybe we can warn or take automatic action (as renaming the files as backup and staring from a fresh db)

Reply:
When "opening" or "repairing" a database it it convert the existing MANIFEST file to tables (other kind of file) and it creates a new manifest file (with a different timestamp in the name that's the reason of the `-\d*`) ALWAYS. The new manifest file is a few kbs always.